styles.css

```diff
@@ -62,13 +62,43 @@ button.collaboration {
   font-weight: 500;
 }
 
-button.collaboration:hover {
+a.see-project {
+  border: 1px solid #fff;
+  background-image: url(./images/Union.png);
+  background-repeat: no-repeat;
+  background-position: center right 0.688rem;
+  width: 11.125rem;
+  margin-top: 1.25rem;
+  padding: 0.625rem;
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
```

Comment:
- [ ] Well done @Zilola-Nazarova 👏🏽 👏🏽 👏🏽 I almost approved at first, just a little detail, kindly remove the underline using `text-decoration: none;` to look similar to the Figma template and a better UI.